.vib/harbor-exporter/goss/harbor-exporter.yaml

```diff
@@ -0,0 +1,25 @@
+group:
+  harbor:
+    exists: true
+user:
+  harbor:
+    exists: true
+file:
+  /var/log/jobs:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+command:
+  # Ensure swagger binary doesn't exist
+  check-swagger-binary:
+    exec: ls /opt/bitnami/harbor-exporter/bin/swagger 2>/dev/null
+    exit-status: 2
```

Comment:
As done for https://github.com/bitnami/containers/pull/30739, I would modify this check to use Goss' `file` directive.